### DISCUSSION TITLE
How can. Net for Apache spark connect to MySQL?

### DISCUSSION THREAD

**Discussion by KawhiWei** [dotnet/spark#750] [Q&A] [answered]
How can. Net for Apache spark connect to MySQL?

**Comment by mwysocki87** [comment]
AFAIK you’ll have to write a thin wrapper around MySQL client JAR-which is the biggest issue stopping me from migrating away from Java...

**Comment by imback82** [comment, marked as answer]
@GeorGeWzw Can you check this thread? https://github.com/dotnet/spark/issues/610#issuecomment-668429774

@mwysocki87 Could you elaborate on the thin wrapper you had to work around?

**Comment by mwysocki87** [comment]
Oh, @imback82, I was under the impression that whenever I want to use some external JAR, I have to write a wrapper around it. If it's not the case, then I stand corrected;)

**Comment by kimmoxter** [comment]
So there is no way to connecto to Mysql? I need to read data from Mongo and insert it on Mysql..

**Reply by imback82** [reply]
Please try https://github.com/dotnet/spark/issues/610#issuecomment-668429774

**Reply by Niharikadutta** [reply]
You can also refer to [this article](https://docs.microsoft.com/en-us/dotnet/spark/how-to-guides/connect-to-sql-server).
